php/insertevent.php

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Sign Up</title>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-T3c6CoIi6uLrA9TneNEoa7RxnatzjcDSCmG1MXxSR1GAsXEV/Dwwykc2MPK8M2HN" crossorigin="anonymous" />
+    <link rel="stylesheet" href="../css/colormodes.css" />
+    <link rel="stylesheet" href="../css/navbar.css" />
+</head>
+<body>
+    <?php
+    include "connect.php";
+
+    $eventName = @$_POST["eventName"];
+    $quota = @$_POST["quota"];
+
+    $inserteventsql = "INSERT INTO events VALUES (null, '$eventName', '$quota' )";
+    $result = mysqli_query($con, $inserteventsql);
+
+    if ($result) {
+        ?>
+            <!-- insert participant response 1 -->
+            <div class="container">
+                <main>
+                    <div class="py-5 text-center">
+                        <h2>Insert Event</h2>
+                        <p>Event successfully insert. Redirecting you to manage event page...</p>
+                    </div>
+                </main>
+            </div>
+        <?php
```

Comment:
Good work on this. Can you add `else` statement in case failed to insert event, and redirect to manage event.